packages/projects/src/components/ProjectForm.jsx

```diff
@@ -296,7 +312,47 @@ const ProjectFormAlert = ({ formStatus, errorMessage, projectHandle }) => {
 const ProjectForm = ({ mode, projectHandle }) => {
   const [state, dispatch] = useReducer(reducer, INITIAL_STATE)
 
-  const { services } = useServices()
+  const { services, currentUser } = useServices()
+  const user = currentUser()
+
+  useEffect(() => {
+    (async () => {
+      const { iss, groupIds } = user
+
+      const { entityFactory } = await services()
+      const groupsApi = await entityFactory({ resource: 'group' })
```

Comment:
I thought it would be confusing to use `groups` here to refer to the API, then `userGroups` below to refer to a collection of groups and not an API, but ok.